presto-main/src/main/java/io/prestosql/event/SplitMonitor.java

```diff
@@ -88,9 +88,9 @@ private void splitCompletedEvent(TaskId taskId, DriverStats driverStats, @Nullab
                                     ofMillis(driverStats.getTotalCpuTime().toMillis()),
                                     elapsedTime,
                                     queuedTime,
-                                    ofMillis(driverStats.getRawInputReadTime().toMillis()),
-                                    driverStats.getRawInputPositions(),
-                                    driverStats.getRawInputDataSize().toBytes(),
+                                    ofMillis(driverStats.getPhysicalInputReadTime().toMillis()),
```

Comment:
There seem to be a bug, see: ```io.prestosql.operator.DriverContext#getDriverStats```:
```
            physicalInputReadTime = inputOperator.getAddInputWall();
            ...
            internalNetworkInputReadTime = inputOperator.getAddInputWall();
```
we shouldn't be adding two of them. Either:
1. there should be two separate fields in `DriverContext` for reporting physical and internal network read times
2. We should remove `internalNetworkInputReadTime` altogether (it's not set by anyone, so it's always 0).

I think we should do 2)